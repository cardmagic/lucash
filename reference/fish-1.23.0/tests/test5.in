
set smurf green

switch $smurf;
	case "*ee*"
		echo Test 1 pass
	case "*"
		echo Test 1 fail
end;

switch $smurf
	case *ee*
		echo Test 2 fail
	case red green blue
		echo Test 2 pass
	case "*"
		echo Test 2 fail
end

switch $smurf
	case cyan magenta yellow
		echo Test 3 fail
	case "?????"
		echo Test 3 pass
end
