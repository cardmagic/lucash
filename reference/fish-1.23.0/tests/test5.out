Test 1 pass
Test 2 pass
Test 3 pass
